HomePage/index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>X Mart | Home</title>
+    <link rel="stylesheet" href="style.css" />
```

Comment:
Suggestion: add meta tags for author and description for better SEO

Reply:
okay